provider/cloudflare/cloudflare.go

```diff
@@ -172,13 +172,20 @@ type RecordParamsTypes interface {
 
 // updateDNSRecordParam is a function that returns the appropriate Record Param based on the cloudFlareChange passed in
 func updateDNSRecordParam(cfc cloudFlareChange) cloudflare.UpdateDNSRecordParams {
-	return cloudflare.UpdateDNSRecordParams{
+	recordParam := cloudflare.UpdateDNSRecordParams{
 		Name:    cfc.ResourceRecord.Name,
 		TTL:     cfc.ResourceRecord.TTL,
 		Proxied: cfc.ResourceRecord.Proxied,
 		Type:    cfc.ResourceRecord.Type,
-		Content: cfc.ResourceRecord.Content,
 	}
+
+	if cfc.ResourceRecord.Type == "SRV" {
```

Comment:
this seems identical to cloudflare.CreateDNSRecordParams. Shell we create a shared method?

Reply:
New lines added, not covered with unit tests
![Screenshot 2025-02-27 at 07 18 54](https://github.com/user-attachments/assets/0e7c364d-a859-4196-bae4-e4cd8d786e9e)